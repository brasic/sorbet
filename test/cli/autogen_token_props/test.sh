#!/bin/bash
set -eu
cd test/cli/autogen_token_props || exit 1

../../../main/sorbet --silence-dev-message -p token-prop-analysis --stop-after=namer . 2>&1

