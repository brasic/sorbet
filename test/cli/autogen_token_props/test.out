No errors! Great job.
TokenPropsModel:
  props:
  - a
  - m
  - tok
  - bc
  - d

